services/learn-card-network/brain-service/src/routes/profiles.ts

```diff
@@ -687,26 +700,49 @@ export const profilesRouter = t.router({
         .input(
             z
                 .object({
+                    expiration: z.number().optional().default(3600), // Default to 1 hour in seconds
```

Comment:
The [learncardapp PR](https://github.com/WeLibraryOS/learncardapp/pull/382) says that we want to default to 1 month, not an hour?
![image](https://github.com/user-attachments/assets/7946f00f-3de5-4fd5-8146-814560a51cc3)

Reply:
fixed - thanks! 🙏